streaming/java/streaming-state/src/test/java/org/ray/streaming/state/keystate/state/impl/OperatorStateImplTest.java

```diff
@@ -0,0 +1,99 @@
+package org.ray.streaming.state.keystate.state.impl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.ray.streaming.state.backend.OperatorStateBackend;
+import org.ray.streaming.state.backend.impl.MemoryStateBackend;
+import org.ray.streaming.state.keystate.desc.ListStateDescriptor;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+
+public class OperatorStateImplTest {
+
+  OperatorStateImpl<Integer> operatorState;
+  ListStateDescriptor<Integer> descriptor;
+  OperatorStateBackend operatorStateBackend;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    String table_name = "kepler_hbase_operator";
+    Map config = new HashMap<>();
+
+    operatorStateBackend = new OperatorStateBackend(new MemoryStateBackend(config));
+
+    descriptor = ListStateDescriptor
+      .build("OperatorStateImplTest" + System.currentTimeMillis(), Integer.class, true);
+    descriptor.setNumber(1);
+    descriptor.setIndex(0);
+    descriptor.setTableName(table_name);
+
+    operatorState = (OperatorStateImpl<Integer>) operatorStateBackend.getSplitListState(descriptor);
+  }
+
+  //@Test
+  public void testInit() throws Exception {
+    operatorStateBackend.setBatchId(1L);
+    // 第一次启动
+    List<Integer> list = operatorState.get();
+    Assert.assertEquals(list.size(), 0);
+
+    for (int i = 0; i < 100; i++) {
+      operatorState.add(i);
+    }
+    Assert.assertEquals(operatorState.get().size(), 100);
+    operatorStateBackend.finish(1L);
+    operatorStateBackend.commit(1L, "");
+    operatorStateBackend.ackCommit(1L, 0);
+
+    operatorStateBackend.finish(5L);
+    operatorStateBackend.commit(5L, "");
+    operatorStateBackend.ackCommit(5L, 0);
+
+    /**
```

Comment:
Inline comments should not be document comment. Also you can use `Assert` for assertion and factor out code block you should add inline explaining document to a dedicated method.

Reply:
Thanks for reminding. I'll fix this.